loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -213,6 +213,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		];
 
 		this._currentDepth = 0;
+		this.helpmacroInnerText = '';
```

Comment:
please revert changes in this file I can do the same in the core. we should avoid changing structure like this in JS code.